.github/workflows/compat-check-latest-latest.yml

```diff
@@ -12,27 +12,32 @@ jobs:
     with:
       lightning-version: latest
       pennylane-version: latest
+    secrets: inherit # pass all secrets
```

Comment:
Any worries about using this? Will it allow secrets to leak to 3rd party workflows?

Reply:
As checked with @rashidnhm, this should be fine here as we own these workflows.